Tests/test_image_transform.py

```diff
@@ -143,6 +143,46 @@ def op(im, sz):
 
         self._test_alpha_premult(op)
 
+    def _test_nearest(self, op, mode):
+        # create white image with half transparent,
+        # with the black half transparent.
+        # do op,
+        # the image should be white with half transparent
+        if mode == "RGBA":
+            transparent = (255, 255, 255, 0)
+            opaque = (255, 255, 255, 255)
+        elif mode == "LA":
+            transparent = (255, 0)
+            opaque = (255, 255)
+        else:
+            assert False, "bad test"
```

Comment:
```suggestion
```
You wouldn't rather avoid adding lines that deliberately don't have coverage? If this method was used with another mode, I feel like it would be hard to miss this conditional block.

Reply:
I put that in to stop my IDE complaining about uninitialized values, but you're right, not a great solution. I replaced the if with a dict lookup.

Reply:
Thanks